tracing/src/main/java/com/palantir/tracing/TraceLocal.java

```diff
@@ -29,14 +29,18 @@
  *
  * A trace local is either set (has a value) or unset (has no value).
  *
- * Outside of a trace (i.e. when {@link Tracer#hasTraceId()} is false) then the trace local is always unset.
```

Comment:
I'm curious why this was removed, I think it provides helpful context to new users.

Reply:
sorry, not sure what happened - reverting